submissions/adeyemiJuwon/css/product.css

```diff
@@ -0,0 +1,307 @@
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@200;300;400&family=PT+Sans:ital@0;1&display=swap');
+
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+
+}
+body {
+    margin: 0;
+    padding: 0;
+    font-size: 22px;
+    line-height: 27px;
+    font-family: oswald PT sans serif;
+}
+/**********************************
+  HEADING
+  ***********************************/
+
+header{
+    display: flex;
+    justify-content: flex-end;
+    background-color: rgb(255, 166, 95);
+    color:black;
+    align-items: center;
+
+}
+ .logo{
+    margin-right:auto;
+}
+.navbar-link{
+    list-style: none;
+}
+.navbar-link li {
+    display: inline-block;
+    line-height: 19px;
+    font-size: 20px;
+    margin-right: 86px;
+    color:black;
+}
+.navbar-link a{
+    text-decoration: none;
+    color: black;
+}
+.navbar-link li a:hover{
+    color: white;
+}
+
+/**********************************
+  PAGE:PRODUCT
+  ***********************************/
+  .product {
+    display: flex;
+    flex-wrap: nowrap;
+}
+.product-img{
+    border: 1px solid #000;
+    max-width: 469px;
+    margin: 50px 50px 0 50px;
+}
+.product img , .product-related img{
+    width: 100%;
+}
+.product-del{
+    margin-left: 50px;
+    margin-top: 50px;
+    padding-top: 14px;
+
+}
+.product-del h3{
+    font-size: 25px;
+    line-height: 30px;
+}
+.product-del h4{
+    font-size: 22px;
+    line-height: 27px;
+    margin:20px 0;
+}
+.product-del input{
+    max-width: 70px;
+    height: 37px;
+    padding-left: 5px;
+}
+.product-del button{
+    width: 170px;
+    height: 47px;
+    border-radius: 20px;
+    border: none;
+    background-color: #f79521;
+    font-size: 22px;
+    line-height: 27px;
+}
+.description{
+    margin-left: 50px;
+}
+.description h3{
+    text-align:left;
+    font-size: 22px;
+    line-height: 27px;
+    margin-top: 23px;
+}
+.description p{
+    text-align:left;
+    margin:8px 0 30px 0;
+    font-size: 20px;
+    line-height: 25px;
+}
+.related-product{
+    background-color: #ebebeb;
+}
+.related-product h3{
+    text-align: center;
+    font-size: 25px;
+    line-height: 30px;
+    padding-top: 30px;
+}
+.product-related{
+    display: flex;
+    justify-content:space-evenly;
+    flex-wrap: wrap;
+    margin-top: 40px;
+    margin-bottom: 35px;
+}
+.product-related a{
+    text-decoration: none;
+}
+.lap-1, .charger, .bag{
+    max-width: 399px;
+    background-color: #fff;
+    margin-bottom: 30px;
+    padding: 20px;
+}
+.lap-1 h5, .charger h5, .bag h5 {
```

Comment:
Targeting all `h5` elements will work the same.

Reply:
Yes
Thank you